### DISCUSSION TITLE
A sidebox with quick information

### DISCUSSION THREAD

**Discussion by raiden-e** [requarks/wiki#5457] [General]
I'd like to see a sidebox for stuff like ingrediants for receipies or other quick information.

Something like the box on the right side in wikipedia articles.

![image](https://user-images.githubusercontent.com/58954125/178030643-9814b552-015a-460e-8710-e5a6bcec22e7.png)

**Comment by NGPixel** [comment]
See https://requarks.canny.io/wiki/p/templates-content-transclusion-infoboxes

It's in the works for v3.
